packages/terra-table/src/TableHeader.jsx

```diff
@@ -1,34 +1,41 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import 'terra-base/lib/baseStyles';
+import classNames from 'classnames/bind';
+import styles from './TableHeader.module.scss';
+
+const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
    * The children passed to the component
    */
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: [],
```

Comment:
Do children props need a default?

Reply:
I think our current pattern is to have a default to any non-required props.

Reply:
Shouldn't the default children argument be undefined or null?

Reply:
Does refCallback need undefined then?

Reply:
Functions callbacks are usually undefined, because their default value would be the same.